application/models/GoogleSheets_Model.php

```diff
@@ -88,16 +88,18 @@ public function highlightRow($row_num, $colour)
      *
      * @param string $email The email of the user to record.
      * @param string $fullName The full name of the user.
+     * @param string @userUpi The UPI of the user. 
+     * @param string @userUid the ID number of the user. 
      * @param string $paymentMethod The type of payment the user is selecting.
      */
-    public function addNewRecord($email, $fullName, $paymentMethod)
+    public function addNewRecord($email, $fullName, $userUpi, $userUid, $paymentMethod)
     {
         $size = $this->getNumberOfRecords() + 2;
         $range = $this->sheetName . '!A' . $size;
 
         // Creating an array for record
         $timestamp = strval(date("d/m/Y h:i:s"));
-        $values = [[$timestamp, $email, $fullName, '','', $paymentMethod]];
+        $values = [[$timestamp, $email, $fullName, $userUpi, $userUid, $paymentMethod]];
```

Comment:
Keep `$userUid` as empty string until we get rid of it – this will require talking with ASPA first though